File: app/src/main/java/com/testapp/testapp/model/utils/SizeChanger.java
package com.testapp.testapp.model.utils;

/**
 * Created by troll on 04.11.2017.
 */

public class SizeChanger {

    public static int zoomOut(int oldSize, int scale){
        return Math.round(oldSize / scale);
    }

    public static int zoomIn(int oldSize, int scale){
        return Math.round(oldSize * scale);
    }
}

File: app/src/main/java/com/testapp/testapp/kotlin/model/utils/SizeChanger.kt
package com.testapp.testapp.kotlin.model.utils

class SizeChanger {

    companion object {
        fun zoomOut(oldSize: Int, scale: Int): Int = Math.round(oldSize.toDouble() / scale).toInt()

        fun zoomIn(oldSize: Int, scale: Int): Int = Math.round((oldSize * scale).toDouble()).toInt()
    }
}